### ISSUE TITLE
Document tuig, tuig-iosys, tuig-pm

### ISSUE THREAD

**Issue by nic-hartley** [nic-hartley/redshell#41]
A lot of the members are documented but not good enough.

- [x] Write READMEs for each one which give a high-level overview of the crate, why you'd use them, where they fit in, etc.
- [x] Decide versioning policy and add that to the READMEs
- [x] Write root-level crate docs explaining in more detail how to use them.
- [x] Ensure everything `cargo doc`s as expected.
- [x] Root-level documentation for every module, including non-public-facing ones
- [x] Ideally document every single item!
- [x] Ensure messages are always called messages, not events
- [ ] ~~Add a lint to CI for things missing documentation~~
- [x] tuig and redshell logos, maybe?

**Comment by nic-hartley** [comment]
Skipped the readme for `redshell` as after #44 this repo's readme will be that crate's readme too

**Comment by nic-hartley** [comment]
I can't find a good way to require documentation for CI so uhhh. Guess I won't!

**Comment by nic-hartley** [comment]
logos in another issue instead, will do that later.
